api/src/main/java/io/opentelemetry/common/Attributes.java

```diff
@@ -44,6 +44,11 @@
 
     @Override
     abstract List<Object> data();
+
+    @Override
+    public Builder toBuilder() {
```

Comment:
Would be good to keep the Labels in sync and add similar functionality there?

Reply:
Labels actually got it implemented first.  :). https://github.com/open-telemetry/opentelemetry-java/blob/7ca698f8452260c8b8da067149444bb01a04da29/api/src/main/java/io/opentelemetry/common/Labels.java#L114